### ISSUE TITLE
how to use nullable attribute in examples?

### ISSUE THREAD

**Issue by sybrands-place** [jormaechea/open-api-mocker#49]
How can I get a `null` example value for a type `string`? (while using OpenAPI 3.0.3)

```
"errorDescription" : {
   "title" : "The errorDescription schema",
   "type" : "string",
   "nullable": true
}
```

When I use the example above, I get `"errorDescription": "string",` as a result

When I add `"default":  null` there's a `[FATAL] Error while mocking schema ...`

(maybe updating `ajv` is a solution? https://ajv.js.org/v6-to-v8-migration.html)

**Comment by jormaechea** [comment]
Hi @sybrands-place, can you provide a minimal OpenAPI schema to reproduce the issue? This way I can find a solution faster.

**Comment by sybrands-place** [comment]
Sure, here's an example:

`{
  "openapi": "3.0.1",
  "info": {
    "title": "Dummy OpenAPI Mock server with nullable String",
    "version": "1.0.0"
  },
  "servers": [
    {
      "url": "http://localhost:8080",
      "description": "Swagger Mock Server URL"
    }
  ],
  "paths": {
    "/dummy": {
      "get": {
        "summary": "Get dummy information",
        "operationId": "getDummyInformation",
        "responses": {
          "200": {
          "description": "OK",
          "content": {
              "application/json": {
                "schema": {
                  "type": "object",
                  "required": [ 
                    "contactpersonResponse",
                    "additionalProperties"
                  ],
                  "properties": {
                    "dummyDescription": {
                      "title": "The dummyDescription schema",
                      "type": "string",
                      "nullable": true
                    },
                    "extraDescription": {
                      "title": "The extraDescription schema",
                      "type": "object",
                      "default": {}
                    }
                  },
                  "example": {
                    "dummyDescription": null,
                    "extraDescription": {}
                  }
                }
              }
            }
          }
        }
      }
    }
  }
}`

In this case it works, but if you remove the example the value for `dummyDescription` becomes "string". Perhaps this is how it should work, that it only works if you provide an example? I tried setting the `default` value to null, but as mentioned before then there's an error message.

**Comment by jormaechea** [comment]
@sybrands-place sorry for the delay, I've been really busy. I'm now working on allowing `null` examples and default values, it should be ready soon.

**Comment by jormaechea** [comment]
Feature released in `v1.10.0`
